main.go

```diff
@@ -102,9 +102,7 @@ func main() {
 	app.Before = func(c *cli.Context) error {
 		// TODO: LoadConfig should validate some basic properties of a valid config,
```

Comment:
I think I can count the number of times I've said this on one hand, but you can remove this TODO since it's now done :smile: